src/views/Arbitrum.js

```diff
@@ -120,6 +121,9 @@ function Arbitrum(props) {
   const [aumPerformanceData, aumPerformanceLoading] = useAumPerformanceData(params)
   const [glpPerformanceData, glpPerformanceLoading] = useGlpPerformanceData(glpData, feesData, params)
 
+  const [minCollectedFees, maxCollectedFees] = useChartDomain(glpPerformanceData, ["performanceLpBtcCollectedFees", "performanceLpEthCollectedFees", "performanceSyntheticCollectedFees"], [80, 180])
```

Comment:
could you use percentage=50 for both please? currently chart shows on different scale. i agree it fees a bit too "zoomed out", i think 50 is a reasonable trade-off

btw maybe something like `padding` instead of `percentage`? sounds a bit clearer to me. not sure though, not critical